hardware/Honeywell.cpp

```diff
@@ -123,19 +127,19 @@ void CHoneywell::Do_Work()
 //
 bool CHoneywell::WriteToHardware(const char *pdata, const unsigned char /*length*/)
 {
+	
 	const tRBUF *pCmd = reinterpret_cast<const tRBUF *>(pdata);
+	
 	if (pCmd->LIGHTING2.packettype == pTypeLighting2)
 	{
 		//Light command
-
-		int nodeID = pCmd->LIGHTING2.id3;
+		
+		int nodeID = pCmd->LIGHTING2.id4;
```

Comment:
Would this ID change cause all existing users to have new devices or cause other issues ?

Reply:
I actually change this id wrong at the previous commit. so this line is a roll back.